script_runner/tasks.py

```diff
@@ -73,7 +79,7 @@ def run(script_path, process=None, ssl_cert_content=None, **kwargs):
     os.chmod(script_path, 0755)
```

Comment:
This line will already raise `OSError(2)`, so is there a need to use `safe_remove_script` in this operation?